### ISSUE TITLE
\i <filename> causes "run() got an unexpected keyword argument 'on_error'"

### ISSUE THREAD

**Issue by mikebissell** [dbcli/pgcli#466]
Anytime I try to import a SQL file using the \i command, I get this error message instead (PostgreSQL 9.3.3, Python 2.7.10, OS X 10.10.5).

Version: 0.20.1
Chat: https://gitter.im/dbcli/pgcli
Mail: https://groups.google.com/forum/#!forum/pgcli
Home: http://pgcli.com
pgdbdev> \i foo.sql
run() got an unexpected keyword argument 'on_error'

**Comment by amjith** [comment]
Sorry about that. This is fixed in master. You can install from master directly using: 

```
pip install -U https://github.com/dbcli/pgcli/archive/master.zip
```

**Comment by mikebissell** [comment]
Thank you.

**Comment by dariodaic** [comment]
Hi,
I'm reporting this cause I see it's still open.
Encountering it as well. Installed via Brew.
Config:
pgcli: 0.20.1
OS X: 10.11.4
PostgreSQL: 9.5.1
Python: 2.7.10
--
\i people.sql
run() got an unexpected keyword argument 'on_error'

**Comment by amjith** [comment]
Did you try installing from master branch? 

```
pip install -U https://github.com/dbcli/pgcli/archive/master.zip
```

I've left the ticket open because we haven't released it yet.

**Comment by dariodaic** [comment]
Works as a charm, excellent.

Thank you!

**Comment by amjith** [comment]
This is now available in v1.0. Please upgrade. 

```
pip install -U pgcli
```
